### DISCUSSION TITLE
UI强调色

### DISCUSSION THREAD

**Discussion by X0980** [Richasy/Bili.Uwp#153] [想法]
现在的强调色都是系统主题的强调色，不看左上角图标都不知道这是b站，云大佬也说过，虽然我们是第三方，但是也不能太放飞自我对吧哈哈，可不可以把选中阴影，按钮这些统一成官方粉色啊，1.0就做了这个，就是不全。。
![image](https://user-images.githubusercontent.com/90809902/133949308-4f89fc52-5df7-4bf3-bb51-e2a325e41e18.png)
![image](https://user-images.githubusercontent.com/90809902/133949318-fa30f32f-c102-4542-ac98-9ec94998d744.png)
![image](https://user-images.githubusercontent.com/90809902/133949350-7ebcc2ae-90f2-4b1b-b190-309d9ff036eb.png)
![image](https://user-images.githubusercontent.com/90809902/133949386-74209e29-ea62-4e49-abb9-fba915334176.png)
![image](https://user-images.githubusercontent.com/90809902/133949399-d5b04a0c-06f1-4dc2-bf04-5854ed3cfb7c.png)
![image](https://user-images.githubusercontent.com/90809902/133949415-ad5ea89f-d3c7-484f-9bd3-df22ad60da4f.png)

**Comment by Richasy** [comment]
已列上日程
